crates/pathfinder/src/storage/state.rs

```diff
@@ -1047,6 +1062,80 @@ impl ContractsStateTable {
     }
 }
 
+/// Stores all known [Starknet state updates][crate::rpc::types::reply::StateUpdate].
+pub struct StarknetStateUpdatesTable {}
+impl StarknetStateUpdatesTable {
+    /// Inserts a StarkNet state update accociated with a particular block into the [StarknetStateUpdatesTable].
+    ///
+    /// Overwrites existing data if the block hash already exists.
+    pub fn insert(
+        tx: &Transaction<'_>,
+        block_hash: StarknetBlockHash,
+        state_update: &StateUpdate,
```

Comment:
So we're using the RPC JSON schema for storage?

That actually seems more volatile than the sequencer schema: https://github.com/starkware-libs/starknet-specs/pull/26

Reply:
Lemme take that discussion back on slack and I'll post the conclusions here and act accordingly.

Reply:
Following a discussion with @kkovaacs:
- both types are volatile (rpc && sequencer) because of incoming `cairo 0.10.0` changes
- spec v 0.1.0 contains `nonces` in the rpc state update already, whereas they're not on the sequencer side yet
- let's keep the rpc type in the db then.